package.json

```diff
@@ -33,6 +33,7 @@
     "@webscopeio/react-textarea-autocomplete": "^4.8.1",
     "axios": "^0.21.2",
     "bs58": "^4.0.1",
+    "chart.js": "^3.9.1",
```

Comment:
chart.js is still being added, couldn't we use highcharts for this? @Adesojisouljay